### ISSUE TITLE
why the OPT model's attention mask have 3 dimensions

### ISSUE THREAD

**Issue by DogeWatch** [NVIDIA/FasterTransformer#392]
the  attention_mask of huggingface OPTmodel only have two dimensions, (batch_size, seq_length)
![企业微信截图_d9886fde-d810-4d09-a2f1-949e8e6b469d](https://user-images.githubusercontent.com/13670813/207260093-c9d41e0b-050d-40b0-b9f5-c7e8e602d48d.png)
but the example of faster transformer make attention_mask 3 dimentions, (batch_size, seq_length, seq_length)
https://github.com/NVIDIA/FasterTransformer/blob/46e1f4a073b1c1eb9a22e712ad4535bde923e36f/examples/pytorch/gpt/utils/gpt_decoder.py#L1126
![企业微信截图_4808e49d-d37c-41e6-bac3-54a3315d2b7d](https://user-images.githubusercontent.com/13670813/207260654-cadfe322-361e-4f71-8d36-12a454c2d29e.png)
so I don't know how to pass the attention_mask generated from tokenizer to the faster transformer opt model

I want only to use the opt model forward function to get the last token hidden state, don't use generate function. Is there any example to show how to do that?
